src/main/java/core/basesyntax/Buldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Buldozer extends Machine{

```

Comment:
There should be a space between the class name 'Buldozer' and the curly brace '{' to follow Java code conventions.